src/_includes/layouts/base.njk

```diff
@@ -29,9 +29,8 @@
     {% include "partials/header.njk" %}
 
     <div class="content-wrapper">
-
-      <main id="mc" class="container-xl mx-auto general-layout flow">
-        {% block content %}{{ content | safe }}{% endblock %}
+      <main id="mc" class="container-xl mx-auto xl:px-0 sm:px-500 px-300 general-layout">
+        {{ content | safe }}
```

Comment:
We will need block back around `{{content | safe }}` to work with the new way we are doing foundation/component documentation pages

Reply:
Oops yes, definitely didn't mean to remove that. I put it back in!.